cellpack/autopack/Ingredient.py

```diff
@@ -5001,11 +5110,24 @@ def jitter_place(
         t1 = time()  # for timing the functions
         insidePoints = {}
         newDistPoints = {}
+
         for attempt_number in range(self.nbJitter):
             packing_location, _, _, _ = self.randomize_translation(
                 env, targeted_master_grid_point, rot_mat
             )
             jitter_rot = self.randomize_rotation(rot_mat, env)
+
+            if env.ingrLookForNeighbours and self.packingMode == "closePartner":
```

Comment:
Add this in for partner packing. But ideally, this is a more general call that doesnt have to be in each different place call.